example/trace/otlpgrpc/example.go

```diff
@@ -31,15 +34,18 @@ import (
 	"google.golang.org/grpc/credentials/oauth"
 )
 
-func initTracer() (func(), error) {
+func initTracer(projectID string) (func(), error) {
 	ctx := context.Background()
 
 	creds, err := oauth.NewApplicationDefault(ctx)
 	if err != nil {
 		panic(err)
 	}
 
-	exporter, err := otlptracegrpc.New(ctx, otlptracegrpc.WithDialOption(grpc.WithPerRPCCredentials(creds)))
+	res := resource.NewWithAttributes(semconv.SchemaURL, attribute.String("gcp.project_id", projectID))
```

Comment:
Can we use OTEL_RESOURCE_ATTRIBUTES instead?

Reply:
yup, updated